pkg/logql/log/label_filter.go

```diff
@@ -179,7 +180,13 @@ func (d *BytesLabelFilter) Process(line []byte, lbs *LabelsBuilder) ([]byte, boo
 }
 
 func (d *BytesLabelFilter) String() string {
-	return fmt.Sprintf("%s%s%d", d.Name, d.Type, d.Value)
+	b := strings.Map(func(r rune) rune {
+		if unicode.IsSpace(r) {
```

Comment:
Ah, you've said that Humanize would add white space between the value and units. Where is this `String()` method used, though? And why would it fail the original version?

Reply:
it is used by the engine to send the query to downstream storages. We re-stringify it because it's the best way to send it on the wire.

Reply:
Ah, thanks for clarifying.